language-examples/nodejs/rest/app.js

```diff
@@ -240,13 +240,10 @@ const port = process.env.PORT || 8080;
 app.listen(port, () => console.log(`Listening on port ${port}...`));
 
 function getJwtToken(auth_url, client_id, client_secret) {
-  const url = `${auth_url}/oauth2/token`;
-  const encoded = Buffer.from(`${client_id}:${client_secret}`).toString(
-    "base64"
-  );
+  const url = auth_url?.endsWith('/oauth2/token') ? auth_url : `${auth_url}/oauth2/token`;
```

Comment:
Nit: Since the suffix is repeated and tightly coupled, we can make it a bit easier to read and maintain by extracting it into a variable:

```ts
const OAUTH_SUFFIX = "/oauth2/token";
const url = auth_url?.endsWith(OAUTH_SUFFIX) ? auth_url : `${auth_url}${OAUTH_SUFFIX}`;
```

Reply:
Is it possible for `auth_url` to end with `"/oauth2/token/"`? This edge case will result in a bug. Should we sanitize `auth_url` by trimming the last character if it's a trailing slash?